src/accordion/AccordionPanel.ts

```diff
@@ -26,30 +27,23 @@ export const useAccordionPanel = createHook<
   compose: unstable_useId,
   keys: ACCORDION_PANEL_KEYS,
 
-  useProps(options, { ref: htmlRef, style: htmlStyle, ...htmlProps }) {
-    const { id, registerPanel, items, activeItems } = options;
-
+  useProps({ id, registerPanel }, { ref: htmlRef, ...htmlProps }) {
     const ref = React.useRef<HTMLElement>(null);
-    const item = items.find(({ panel }) => panel?.id === id);
-    const buttonId = item?.button?.id;
-    const isOpen = item ? activeItems.includes(item.id) : false;
 
     React.useEffect(() => {
       if (!id) return undefined;
 
       registerPanel?.({ id, ref });
     }, [id, registerPanel]);
 
-    const style = {
-      display: `${isOpen ? "block" : "none"}`,
-      ...htmlStyle,
-    };
+    const { item, isOpen } = useAccordionItemContext();
+    const buttonId = item?.button?.id;
 
     return {
       role: "region",
       "aria-labelledby": `${buttonId ? buttonId : undefined}`,
```

Comment:
It will be undefined if it's not found anyways right? so no need to check if it's undefined and then fallback to undefined.

Reply:
Oh, my bad. I changed it in Panel component but forgot here.
Point !!